core/workspace.js

```diff
@@ -217,6 +283,9 @@ Blockly.Workspace.prototype.clear = function() {
   while (this.topBlocks_.length) {
     this.topBlocks_[0].dispose();
   }
+  while (this.topComments_.length) {
+    this.topComments_[this.topComments_.length - 1].dispose();
```

Comment:
I noticed that blocks are disposed starting at the beginning of the list and comments are disposed starting at the end of the list. Are there pros and cons here?

Reply:
Doing it from the end should avoid a bunch of array shuffling.

Reply:
yeah, that was my assumption. Should blocks be switched to doing it from the end as well?